per_domain.rego

```diff
@@ -0,0 +1,21 @@
+package policy
+
+import future.keywords.in
+import data.phylum.domain
+import data.phylum.level
+
+issue["risk level cannot exceed medium"] {
```

Comment:
What do you think about this variation?

```rego
issue contains "risk level cannot exceed medium" if {
    data.issue.domain in {domain.AUTHOR, domain.ENGINEERING, domain.VULNERABILITY}
    data.issue.severity > level.MEDIUM
}
```

It requires `import future.keywords.contains` `import future.keywords.if` (Or maybe only one of them. One might include the other.). Doing it this way might read better to users that are new to rego.

Reply:
It turns out that `opa fmt` will add these for you if you just import the keywords. :+1: